megalinter/descriptors/spell.megalinter-descriptor.yml

```diff
@@ -3,6 +3,8 @@ descriptor_type: other
 descriptor_flavors:
   - all_flavors # Applicable to CI in any language project
   - cupcake
+file_extensions:
```

Comment:
I want default behaviour to remain the current one :)
So please don't update the descriptor ^^

Reply:
But if I don't update the descriptor, neither the schema nor the documentation will be updated to reflect that the SPELL_*_FILE_EXTENSIONS parameter can be set, right?

Reply:
You're right :)
To do that, you must remove the condition here :)
https://github.com/bdovaz/megalinter/blob/d03b3efdcb48eefbd813dce5f5628705f6cbc664/.automation/build.py#L1255

Reply:
Done

Reply:
I just realized that even removing that second condition won't work because the value of "lint_all_other_linters_files" is set to true: https://github.com/bdovaz/megalinter/blob/main/megalinter/descriptors/spell.megalinter-descriptor.yml

The condition in the script you mentioned compares it to false